### PR TITLE
Project template - Documentation edited

### PR DESCRIPTION
**Pull request by alexander-levitski**

### CODE REVIEW

File: community/cloud-foundation/templates/project/README.md [hunk 1]
```diff
@@ -1,86 +1,60 @@
...
-1.  **Create/find a *Billing Account* associated with the organization**
-
-    * See: https://cloud.google.com/support/billing/
-    * Take note if the Billing Account ID that looks like `00E12A-0AB8B2-078CE8`
-
-1.  **Give the *DM Service Account* the following permissions on the Billing Account:**
-    * `roles/billing.user`
-        * This is visible in Cloud Console's IAM permissions in
-          *Billing -> Billing Account User*.
+7.  Create/find the *Billing Account* associated with the Organization. See: https://cloud.google.com/support/billing/. Take note of the *Billing Account*'s ID, which is formatted as follows:`00E12A-0AB8B2-078CE8`.
```
Comment (fawix) [inline community/cloud-foundation/templates/project/README.md]: Just making sure.. this is a fake billing account right?  should we make something more obviously fake like like "012345-6789AB-CDEF01"  ?

Comment (fawix) [reply community/cloud-foundation/templates/project/README.md]: Note: this comment also applies to line 51 of project.py.schema

Comment (sourced-vince) [reply community/cloud-foundation/templates/project/README.md]: @fawix This is a fake billing account.

Comment (fawix) [reply community/cloud-foundation/templates/project/README.md]: ok, I'm good then :)

**Pull request merged**